### DISCUSSION TITLE
Redux DevTools + Electron + Vite

### DISCUSSION THREAD

**Discussion by Stanzilla** [reduxjs/redux-devtools#1693] [Q&A]
Anyone here had success getting the above mentioned combo to work? So far I can only get the extension to load when the app is packaged, it never works when the app is in dev mode.

**Comment by pablocordova** [comment]
I have the same problem, any solution?

**Comment by TaoXiong05** [comment]
try edge or other bs, it has been a problem with chrome since last yaer
